samples/parselib/src/main/cpp/wav/WavStreamReader.cpp

```diff
@@ -52,11 +52,10 @@ int WavStreamReader::getSampleEncoding() {
                 return AudioEncoding::PCM_16;
 
             case 24:
-                // TODO - Support 24-bit WAV data
-                return AudioEncoding::INVALID; // for now
+                return AudioEncoding::PCM_24;
 
             default:
-                return AudioEncoding::INVALID;
+                return AudioEncoding::PCM_32;

```

Comment:
Could add a "case 32" for PCM_32.
If mSampleSize is something weird, like 64, then we could return INVALID for default case.

Reply:
Thanks! Done!